src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -16,7 +17,7 @@ import scala.util.Try
 
 object Backend {
   lazy val LocalBackend = new LocalBackend
-  lazy val JesBackend = new JesBackend { JesConf } //forces configuration resolution to fail now if something is missing
+  lazy val JesBackend = new JesBackend with JesAuthenticationSingleton { JesConf } //forces configuration resolution to fail now if something is missing
```

Comment:
ok, but can the definition of the `JesBackend` class extend `JesAuthenticationSingleton` rather than having this throughout the code?